### PR TITLE
add - semantic support

### PR DESCRIPTION
**Pull request by khemrajrathore**
add - semantic support
fix - https://github.com/Privado-Inc/privado-core/pull/111

### CODE REVIEW

**Comment by khemrajrathore** [comment]
@HiteshMah-Jan  Merge this pull request after merging - https://github.com/Privado-Inc/privado-core/pull/111

**Review by HiteshMah-Jan — APPROVED** [review]

**Comment by pandurangpatil** [comment]
@HiteshMah-Jan https://github.com/Privado-Inc/privado-core/pull/111 this pull request has been merged.

**Pull request merged**
